src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Patch_V1Compat.txt

```diff
@@ -41,3 +44,238 @@ Errors: Error 13-143: The value passed to the 'Patch' function cannot be changed
     {Properties:{Moon:{Name:"Phobos"}}},
     {Planet:"Jupter"})
 Errors: Error 13-251: The value passed to the 'Patch' function cannot be changed.
+
+>> Patch(t1, r1, r_empty);
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

Can you add a test before this one just with `t1` - that will make it easier to know what was the initial value just by looking at the test.

Reply:
Same with `r1` and `r2`

Reply:
Added at the beginning.